packages/core/src/Platform/index.ts

```diff
@@ -1,30 +1,30 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
+import { CustomUserAgent } from '../types/types';
```

Comment:
I'm not sure why this folder is structured the way it is but we should really either:
1. Just get rid of the folder - leaving a `types.ts` on the `src` root
2. Get rid of `index.ts` and just rename the current `types.ts` to `index.ts`

Having an index whose sole purpose to to re-export another file seems pretty pointless.

Even if we don't want to make that change in this PR, I would suggest you update this to:
```suggestion
import { CustomUserAgent } from '../types';
```